ant-project/build.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<project xmlns:ivy="antlib:org.apache.ivy.ant" name="ant-sample" default="build" basedir=".">
+<project xmlns:ivy="antlib:org.apache.ivy.ant" name="ant-sample" default="checkstyle" basedir=".">
```

Comment:
There is no build target, so I switched it to `checkstyle`. Technically it can be run with just `ant` now.